### PR TITLE
GAWB-1068: keep attrs-list updated in state when navigating through workspace tabs

### PR DESCRIPTION
**Pull request by davidangb**
This feels like a pretty brute-force fix, and I'm not super-proud of the duplication between lines 222 and 224. Thoughts? It does seem to fix the bug ... 
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  - Acceptance criteria exists and is met
  - Note any changes to implementation from the description
  - Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  - Authentication
  - Authorization
  - Encryption
  - Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- Review cycle:
  - LR reviews
  - Rest of team may comment on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH
  - Submitter updates documentation as needed
  - Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs [hunk 1]
```diff
@@ -227,6 +227,8 @@
    :refresh
    (fn [{:keys [props state]}]
      ((:request-refresh props))
+     (swap! state assoc :attrs-list
+            (vec (dissoc (get-in props [:workspace "workspace" "attributes"]) "description")))
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs]: The refresh is async, so there's a chance that the attributes have changed and this is inaccurate.

Comment (davidangb) [reply src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs]: after in-person discussion, moved this call to :component-did-mount.

**Comment by MattPutnam** [comment]
👍

**Comment by dmohs** [comment]
👍

**Pull request merged**